packages/three-vrm-core/src/expressions/index.ts

```diff
@@ -8,3 +8,6 @@ export { VRMExpressionMorphTargetBind } from './VRMExpressionMorphTargetBind';
 export { VRMExpressionOverrideType } from './VRMExpressionOverrideType';
 export { VRMExpressionPresetName } from './VRMExpressionPresetName';
 export { VRMExpressionTextureTransformBind } from './VRMExpressionTextureTransformBind';
+
+import type * as V0VRM from '@pixiv/types-vrm-0.0';
+export type { V0VRM };
```

Comment:
If I understand correctly, these two lines are needed to control whether we are going to expose `V0VRM` from three-vrm-core or not, right?

Why we need to have this in `src/expressions/index.ts` , not `src/index.ts` ?